assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_new_edit_assignment.vm

```diff
@@ -751,7 +751,7 @@
 					</div>
 
 					<!-- Start TII custom options -->
-					#if($reviewServiceName.equals("Turnitin"))
+					#if($reviewServiceProviderId == 199481773)
```

Comment:
These values should be defined AssignmentConstants

Reply:
@baholladay can you leave a comment here to address MAtt's concern. Matt can you remove your block once you are no longer concerned.